src/App.js

```diff
@@ -92,7 +98,7 @@ function App() {
     );
   }
 
-  //Adding a phrase 
+  //Adding a phrase
```

Comment:
Since you think it is necessary to structure your App.js with comments, you might think about a `usePhrase` custom hook in order to minimize your App ;)

Reply:
Hey Paul thx ;) It´s a really good idea...I keep it in mind when I´ll do a refactoring ;)